core/templates/dev/head/pages/error/error.html

```diff
@@ -67,10 +67,8 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
-    @media (max-width: 768px) {
-      .oppia-footer {
-        position: relative;
-      }
+    .oppia-wide-panel-content {
+      margin-bottom: 15em
```

Comment:
@shaz13 Which resolution did this break at? I tested it on all the available resolutions that I have on my system.